sleap/gui/color.py

```diff
@@ -177,6 +195,22 @@ def get_track_color(self, track: Union[Track, int]) -> ColorTupleType:
             return (0, 0, 0)
 
         return self.get_color_by_idx(track_idx)
+    
+    def get_instance_group_color(self, instanceGroup: Union[InstanceGroup, int]) -> ColorTupleType: 
```

Comment:
`snake_case` please!
```suggestion
    def get_instance_group_color(self, instance_group: Union[InstanceGroup, int]) -> ColorTupleType: 
```